compiler/x/codegen/X86Ops.hpp

```diff
@@ -167,6 +166,36 @@ inline TR_X86OpCodes SizeParameterizedOpCode(bool is64Bit =
 #define MOVSMemReg     SizeParameterizedOpCode<MOVSDMemReg     , MOVSSMemReg     >
 #define MOVSRegMem     SizeParameterizedOpCode<MOVSDRegMem     , MOVSSRegMem     >
 
+// Aliases to Jcc
```

Comment:
Hmm isn't defining these a bit dangerous since they are global replacements of one string with another? It might make finding uses of the opcode in future harder...

Reply:
Good point, updated.